.circleci/config.yml

```diff
@@ -70,7 +73,7 @@ jobs:
 workflows:
   lint-and-test:
     jobs:
-      - lint
+      - lint 
```

Comment:
Is the base for this one develop by mistake? 🤔 (repeating)

Reply:
GitHub automatically switches branches when the base branch is merged. `fix/update-lerna-and-fix-unlink-call` was the base branch for all PRs